### PR TITLE
Added recipe 'cucumber gin'

### PR DESCRIPTION
**Pull request by JoshuaMart**

### CODE REVIEW

File: src/recipes/cucumber-gin.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            "quantity": "3",
+            "measure": "Strips",
+            "ingredient": "Fresh cucumber"
+        },
+        {
+            "quantity": "8",
+            "measure": "cl",
+            "ingredient": "mint syrup"
+        }
+        ,
```
Comment (alfg) [inline src/recipes/cucumber-gin.json:31]:
```suggestion
        },
```

**Review by alfg — CHANGES_REQUESTED** [review]
One minor suggestion. Thanks!

**Comment by JoshuaMart** [comment]
Well seen :)

**Comment by alfg** [comment]
Thanks!

**Pull request merged**